### PR TITLE
Basic AHelp Panel, Ported & Fixed

### PR DESCRIPTION
**Pull request by 20kdc**
## About the PR

Continuation of https://github.com/space-wizards/space-station-14/pull/3049

*But this time it works!*

Full explaination:

Each player has an AHelp channel, which they can always receive messages on, and they can always send messages on.
All admins receive and can send messages on all ahelp channels.

When an ahelp message is received, if a window does not exist or is hidden for that channel, it is created/shown.

To be absolutely clear, this PR is not intended to be some sort of full ticketing system. Frankly I suspect such a thing would be such a big PR that it would be a nightmare to keep it up to date with the way things change around here.

It is meant to be an admin/player semi-private chat system, since that's simple to implement and is what is needed (see: OOC being misused for this)

**Screenshots**
image

**Changelog**

:cl:
- add: 'AHelp' next to the options menu.

### CODE REVIEW

**Comment by 20kdc** [comment]
there's already a players tab, and if that allowed selection it seems like these should really go there, but, well, it doesn't, they don't
Player Actions Panel, maybe?

**Comment by mirrorcult** [comment]
yeah that seems fine

**Comment by ike709** [comment]
> To be absolutely clear, this PR is not intended to be some sort of full ticketing system. Frankly I suspect such a thing would be such a big PR that it would be a nightmare to keep it up to date with the way things change around here.

Here you go: https://github.com/space-wizards/space-station-14/pull/3881

**Comment by mirrorcult** [comment]
this PR probably needs a lot of discussion from maintainers considering anim tooling is a focus + we still want to potentially reuse ike/PJB's ticketing PR and we'd have to integrate that with this

**Comment by mirrorcult** [comment]
speak of the devil

**Comment by SweptWasTaken** [comment]
Seems p ok for now

**Review by SweptWasTaken — APPROVED** [review]

**Comment by mirrorcult** [comment]
personally im fine w/ this being merged especially since we've been getting players recently and moony requested it

**Comment by moonheart08** [comment]
yea honestly even in a barebones state we need ahelp asap

**Comment by ShadowCommander** [comment]
If the code is fine, then it'd be good to have something, even if we have to replace it later. I'll review this PR later in the week unless someone else gets to it first.

**Review by Zumorica — CHANGES_REQUESTED** [review]
This isn't getting merged until it has the classic `bwoink` sound.

File: Content.Client/Administration/BwoinkSystem.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+namespace Content.Client.Administration
+{
+    [UsedImplicitly]
+    public class BwoinkSystem : SharedBwoinkSystem
+    {
+        [Dependency] private readonly IPlayerManager _playerManager = default!;
+        private readonly Dictionary<NetUserId, BwoinkWindow> _activeWindowMap = new();
+
+        protected override void OnBwoinkTextMessage(BwoinkTextMessage message, EntitySessionEventArgs eventArgs)
```
[the remark this answers is not in the document]
Comment (Zumorica) [reply Content.Client/Administration/BwoinkSystem.cs:23]: I looked into this, this is a problem with many windows in the codebase right now...

[the remark this answers is not in the document]
Comment (Zumorica) [reply Content.Client/Administration/BwoinkSystem.cs:23]: On closer inspection, it seems the GC correctly handles cleaning closed windows up, so it should be fine.

**Pull request merged**